File: src/main/java/net/csibio/aird/bean/MobiInfo.java
/*
 * Copyright (c) 2020 CSi Biotech
 * Aird and AirdPro are licensed under Mulan PSL v2.
 * You can use this software according to the terms and conditions of the Mulan PSL v2.
 * You may obtain a copy of Mulan PSL v2 at:
 *          http://license.coscl.org.cn/MulanPSL2
 * THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT, MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
 * See the Mulan PSL v2 for more details.
 */

package net.csibio.aird.bean;

import lombok.Data;

@Data
public class MobiInfo {

  /**
   * start position in the aird for mobi array
   */
  long dictStart;

  /**
   * end position in the aird for mobi array
   */
  long dictEnd;

  /**
   * ion mobility unit
   */
  String unit;

  /**
   * ion mobility type, see MobilityType
   */
  String type;
}

File: CSharpSDK/Bean/MobiInfo.cs
/*
 * Copyright (c) 2020 CSi Studio
 * AirdSDK and AirdPro are licensed under Mulan PSL v2.
 * You can use this software according to the terms and conditions of the Mulan PSL v2. 
 * You may obtain a copy of Mulan PSL v2 at:
 *          http://license.coscl.org.cn/MulanPSL2 
 * THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT, MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.  
 * See the Mulan PSL v2 for more details.
 */

namespace AirdSDK.Beans
{
    public class MobiInfo
    {
        public long dictStart; // start position in the aird for mobi array
        public long dictEnd; //end position in the aird for mobi array
        public string unit; //ion mobility unit 
        public string type; //ion mobility type, see MobilityType
    }
}